CHANGELOG.md

```diff
@@ -2,464 +2,292 @@
 
 All notable changes to this project will be documented in this file.
 
-## Unreleased - as of Sprint 22 end 2015-04-20
+## Unreleased - as of Sprint 23 end 2015-05-11
 
-### [Added](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+22+Ending+Apr+20%2C+2015%22+label%3Aenhancement)
+### [Added](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+23+Ending+May+11%2C+2015%22+label%3Aenhancement)
 
-- Providers
-  - Orchestration: Support for Retirement
-  - SCVMM: Provisioning from template, Kerberos Authentication
-  - Amazon: Updated Instance Types (D2 and G2)
-  - Containers: UI updates, refresh button
-  - OpenStack: Autoscale compute nodes
-  - Foreman
-     - Added tag processing during provisioning
-     - Added inventory collection of direct and inherited host/host-group settings
-- UI: Tagging Support for Foreman
-- Rest API: Enhancement to /api/providers to support new provider class
-
-### [Changed](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+22+Ending+Apr+20%2C+2015%22+label%3Aenhancement) 
-
-- Automate
- - Dynamic list refactor
- - Read only fields
- - Detection of User vs. System initiated retirement
- - Update provisioning/retirement entry point in a catalog item or bundle
-- Authentication
-  - Update authentication status when clicking validate for existing Providers
-- UI
-  - IPv6: Hostname changes, IP address removal
-  - SCVMM UI updates
-- Base Appliance
-  - cloud-init rpm added (but there are root ssh login issues)
-  - iptables configured via kickstart 
+- Foreman
+  - Filtering of available Configuration Profiles based on selected Configured Systems in provisioning dialog
+  - Exposed additional properties to reporting
+- Automate: Ability to override request description with user message
 
-### [Removed](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+12+Ending+Apr+20%2C+2015%22+label%3A%22technical+debt%22)
+### [Changed](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+23+Ending+May+11%2C+2015%22+label%3Aenhancement) 
 
-- SmartProxy: removed from UI
-- IP Address Form Field: Removed from UI (use Hostname)
+- UI: Foreman Enhancements
 
 
-## Unreleased - as of Sprint 21 end 2015-03-30
+# Botvinnik Release
 
-### [Added](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+21+Ending+Mar+30%2C+2015%22+label%3Aenhancement)
+### Added Features
 
-- Providers: OpenStack
+#### Providers
+- General
+  - Added refresh status and errors, viewable on Provider Summary Page.
+  - Added collection of raw power state and exposed to reporting.
+  - Orchestration: Support for Retirement
+  - Update authentication status when clicking validate for existing Providers
+  - Hosts and Providers now default to use the hostname column for communication instead of IP address.    
+- SCVMM
+  - Provisioning from template, Kerberos Authentication
+  - Virtual DVD drives for templates
+- Containers: UI updates, refresh button
```

Comment:
There are no other containers in this release, so this can all go under Kubernetes.